scripts/extensions/sams/src/components/sets/setListGroup.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {ISetItem, IStorageDestinationItem} from '../../interfaces';
+
+import {
+    ListHeader,
+    ListItemGroup,
+    ListItem,
+    ListItemColumn,
+    ListItemRow,
+} from '../../ui/';
+
+import {SetListItem} from './setListItem';
+
+interface IProps {
+    title: string;
+    noItemTitle: string;
+    marginTop?: boolean;
+    sets: Array<ISetItem>;
+    storageDestinations: {[key: string]: IStorageDestinationItem};
+    onItemClicked(set: ISetItem): void;
+    onDelete?(set: ISetItem): void;
+    onEdit(set: ISetItem): void;
+    currentSetId?: string;
+}
+
+export class SetListGroup extends React.PureComponent<IProps> {
+    render() {
+        return (
+            <React.Fragment>
+                <ListHeader title={this.props.title} marginTop={this.props.marginTop} />
+                <ListItemGroup spaceBetweenItems={true}>
+                    {this.props.sets?.length === 0 ? (
+                        <ListItem inactive={true} shadow={2} noHover={true}>
```

Comment:
There's a shorthand where you don't need to specify the value of a prop if it is `true`.

```<ListItem inactive noHover shadow={2}>```

Reply:
Either way is an acceptable definition of an attribute. I prefer to clearly define the value of the attribute